app/src/main/java/com/onelist/ui/MainActivity.kt

```diff
@@ -87,7 +86,7 @@ class MainActivity : ComponentActivity() {
                 }
             )
             {
-                Text(text = "Submit")
+                Text(text = stringResource(id = R.string.submit))
```

Comment:
It's best practice to have reusable string text referenced in strings.xml than to hard code them in the code. You can support more languages, reuse the same strings across components, activities, etc., and there is many styling and formatting options with strings.xml resources. https://developer.android.com/guide/topics/resources/string-resource#String

Reply:
Definitely.